src/request/src/network.js

```diff
@@ -425,7 +425,7 @@ export class KinveyRequest extends NetworkRequest {
                 properties: this.properties
               });
 
-              return refreshMICRequest.execute()
+              return refreshMICRequest.execute(false, false)
```

Comment:
The first parameter is used to return the raw response rather then just the data from the response. The second parameter is used to retry the request in the event a `401` status code is encountered.